src/main.ts

```diff
@@ -93,9 +119,24 @@ const command: Command = {
 		});
 	},
 	run(helper: Helper, args: TestArgs) {
+		/* istanbul ignore next */
+		function unhandledRejection(reason: any) {
+			console.log('Unhandled Promise Rejection: ');
+			console.log(reason);
+		}
+
+		process.on('unhandledRejection', unhandledRejection);
+
 		return new Promise((resolve, reject) => {
 			if (!helper.command.exists('build')) {
-				reject(Error('Required command: \'build\', does not exist. Have you run npm install @dojo/cli-build?'));
+				reject(Error(`Required command: 'build', does not exist. Have you run 'npm install ${CLI_BUILD_PACKAGE}'?`));
+			}
+			try {
+				const projectName = require(path.join(process.cwd(), './package.json')).name;
+				console.log(blue.bold(`\n▶ Building "${projectName}":`) + `\n`);
```

Comment:
This output with the blue and the arrow is unlike any of the other cli output headings which simply use white underlined text for headings and a single space for indentation. Do you think we should keep this output inline with others?
![screenshot 2017-03-31 12 22 59](https://cloud.githubusercontent.com/assets/814453/24548410/dff6c030-160c-11e7-8769-3ff5cdeb925d.png)